.github/CODEOWNERS

```diff
@@ -21,6 +21,7 @@
 /sdk/platform/http_client/  @vhvb1989
 /sdk/platform/win32/        @antkmsft @jeffreyrichter
 /sdk/storage/               @gilbertw
+/sdk/iot/                   @ericwol-msft @jspaith @ewertons @hihigupt @CIPop @danewalton
```

Comment:
```suggestion
/sdk/iot/                   @CIPop @danewalton @ericwol-msft @ewertons @hihigupt @jspaith
```

Reply:
But up to you, this is really minor thing of no technical difference.

Reply:
good rec. updated

Reply:
Thank you Dane! I'll complete your PR soon as CI/CD finish rebuilding.

Reply:
just to remove an interpretation of hierarchy in the listed names. good suggestion.